axe.d.ts

```diff
@@ -241,9 +243,35 @@ declare namespace axe {
     helpUrl: string;
     tags: string[];
   }
+  interface SerialDqElement {
+    source: string
+    nodeIndexes: number[]
+    selector: number[],
+    xpath: number[],
+    ancestry: number[],
+  }
+  interface PartialRuleResult {
+    id: string,
+    result: 'inapplicable',
+    pageLevel: boolean,
+    impact: null,
+    nodes: object[],
```

Comment:
The `object` type is extremely difficult to use/not what we want.

```suggestion
    nodes: Array<Record<string, unknown>>,
```